Product/Production/Gateway/DocumentQuery_30/src/main/resources/docquery/_30/webservices.xml

```diff
@@ -115,12 +115,12 @@
             <ref bean="TransactionHandler" />
         </jaxws:handlers>
     </jaxws:endpoint>
-    
+
 
 
     <jaxws:endpoint xmlns:edqs="urn:gov:hhs:fha:nhinc:dq:entitydeferredresponsesecured" id="EntityDocQueryDeferredResultSecured"
         address="/EntityService/EntityDeferredResultOptionResponseSecured" serviceName="edqs:EntityDocQueryDeferredResultSecured" endpointName="edqs:EntityDocQueryDeferredResultSecuredPortSoap"
-        implementor="gov.hhs.fha.nhinc.docquery.deferred.entity.EntityDeferredResultsOptionSecured"
+        implementor="gov.hhs.fha.nhinc.deferredresults.entity.EntityDeferredResultsOptionSecured"
```

Comment:
Should have "implementorClass" set to the interface to explicitly state the class type for injection